rappi4/src/constantes/url.js

```diff
@@ -0,0 +1,7 @@
+export const BASE_URL = "https://us-central1-missao-newton.cloudfunctions.net/";
+
+export const HEADERS = {
+  headers: {
+    auth: localStorage.getItem("token"),
+  },
+};
```

Comment:
Essa separação não é recomendada, porque o objeto HEADERS é carregado uma vez no início da aplicação e depois não se atualiza. A opção mais simples é configurar o header na hora de enviar a requisição mesmo 👍 

Ótima tentativa, no entanto!